asapdiscovery-data/asapdiscovery/data/scripts/separate_ligand_script.py

```diff
@@ -0,0 +1,150 @@
+"""
+Script to convert a single pdb file with multiple ligands all named LIG in
+one crystal structure from Diamond to multiple pdb file each containing one
+ligand named LIG bound to the original protein crystall structure and a sdf
+file that contains all of the ligands. All the pdb files would be within a
+directory within the directory of the original protein called output.
+The sdf file containing all the ligands in original file will be also in that
+directory named Combined_ligands.sdf.
+
+Minimal example usage:
+python cdd_to_schema.py \
+-i multiple_ligand_bound.pdb \
+-d directory/
+"""
+
+import argparse
+import os
+from pathlib import Path
+
+import networkx as nx
+from asapdiscovery.data.openeye import (
+    combine_protein_ligand,
+    load_openeye_pdb,
+    save_openeye_pdb,
+    save_openeye_sdfs,
+)
+from pymol import cmd
+
+
+# Arguments define the input file and the output file
+def get_args():
+    parser = argparse.ArgumentParser(description="")
+
+    parser.add_argument(
+        "-d", required=True, help="Directory of input and output files."
+    )
+    parser.add_argument("-i", required=True, help="PDB file input.")
+
+    return parser.parse_args()
+
+
+# The function to isolate ligands in original file and separate out individual ligands
+# Each ligand saved in individual pdb file
+def split_ligands(directory, input_file):
+    # Where original file is from
+    local_path = Path(directory)
+    # Where ending strucutures will be put into
+    os.mkdir(directory + "output/")
+    output_path = Path(directory + "output/")
```

Comment:
You can just do 

```suggestion
    output_path = Path(directory) / "output"
    output_path.mkdir()
```
and then you can get rid of the `os.mkdir` on L48